OrbitGl/App.cpp

```diff
@@ -118,6 +118,11 @@ OrbitApp::OrbitApp(ApplicationOptions&& options,
 }
 
 OrbitApp::~OrbitApp() {
+  AbortCapture();
+
+  process_manager_->ShutdownAndWait();
+  thread_pool_->ShutdownAndWait();
```

Comment:
are these noexcept?

Reply:
No. Should I add a catch-all block? Maybe that's better but this would change behaviour. There was nothing that caught a potential exception before.

If these threw, this would end up in an abort since the destructor is noexcept by default.

Reply:
I did not know d-tors are noexcept by default. I am thinking it might be better to have an explicit catch all -> FATAL block,

Reply:
Added that. PTAL.